autoload/dotoo/parser/headline.vim

```diff
@@ -214,8 +216,13 @@ function! dotoo#parser#headline#new(...) abort
         \ }
 
   if has_key(token, 'type')
+    let blanks = []
     while len(tokens)
       let token = remove(tokens, 0)
+      if token.type != s:syntax.blank.type
+        call extend(headline.content, blanks)
```

Comment:
Euhm, my first idea was to only "save" the blank lines in the text. But, then I realized that, because of the way serialization currently works, in fact all blank lines need to be added, and not just those in the actual content, but the ones at the end, which don't really matter. However, doing it like this will butt all blank lines from metadata at the end of the content.

I would say the situation before my last commit was better, saving the blank lines, but only those in the actual content. But, then, we should keep track of the number of lines, instead of calculating them based on the serialization. Then, all empty lines after the content would be disregarded, while keeping those in the text intact.

To finally answer your question: we don't. It's useless, as this is done BEFORE testing if this is a new headline, and thus the blans will always be added.